File: app/src/main/java/com/arnaudpiroelle/manga/service/ServiceBootReceiver.java
package com.arnaudpiroelle.manga.service;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;

public class ServiceBootReceiver extends BroadcastReceiver {

    @Override
    public void onReceive(Context context, Intent intent) {
        Log.i("ServiceBootReceiver", "AlertManager " + intent.getAction());
        if (intent.getAction().equals("android.intent.action.BOOT_COMPLETED")) {

            DownloadService.updateScheduling(context);
        }
    }
}

File: app/src/main/kotlin/com/arnaudpiroelle/manga/service/ServiceBootReceiver.kt
package com.arnaudpiroelle.manga.service

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.util.Log

class ServiceBootReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        Log.i("ServiceBootReceiver", "AlertManager " + intent.action)

        if (intent.action == "android.intent.action.BOOT_COMPLETED") {
            DownloadService.updateScheduling(context)
        }
    }
}
